### ISSUE TITLE
Compile fails under ubuntu 18.04

### ISSUE THREAD

**Issue by bumm13** [vpinball/vpinball#1670]
After upgrading cmake to 3.25.2 version, I tried using the included "external.sh" script file to install Visual Pinball. It was chugging along fine for quite a while in the terminal until the build process died with the pertinent text being:

CMakeFiles/zedmd_test.dir/build.make:97: recipe for target 'zedmd_test' failed

I've attached a partial log where the compile was working up until it failed. I'm not a programmer so I have no idea what went wrong with the build.
[vpinball-log01.txt](https://github.com/vpinball/vpinball/files/15024416/vpinball-log01.txt)

**Comment by francisdb** [comment]
May I ask why you're trying to build on this very old/[no longer supported](https://ubuntu.com/blog/18-04-end-of-standard-support) Ubuntu version? 

> 18.04 LTS machines will not receive any security updates after 31 May 2023.

**Comment by toxieainc** [comment]
@bumm13 ping  :)

**Comment by bumm13** [comment]
I am on an x86 system. I have since upgraded my Ubuntu install to 22.04.4 LTS and have 3.29.2 version of cmake. Should I open a new issue or stay on this thread?

**Comment by jsm174** [comment]
The ci is building fine under 22. Did you do a clean install of 22 or an upgrade?

**Comment by bumm13** [comment]
It's a clean install (but certainly an upgrade, too). It's still not building but the compile got much further and the error is different.

**Comment by francisdb** [comment]
Can you share the logs?

**Comment by bumm13** [comment]
Well... no need, as the program (after the recent compile attempt, which asked for either nasm or yasm), built fine according to the instructions. I tested both a test table and a different table (.vpx files) and it works great! Sorry for any difficulty I caused; I've had some strange things occur lately with my computer setup so I was a bit anxious. Thank you so much for all the help; I am absolutely amazed and happy with the result of the standalone vpinball under Linux!
